tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py

```diff
@@ -18,13 +18,35 @@
 
 import numpy as np
 
-from ..representation import QuestionAnsweringAnnotation
+from ..representation import QuestionAnsweringAnnotation, ExtendedQuestionAnsweringAnnotation
 from ..utils import read_json
 from ..config import PathField, NumberField, BoolField
 
 from .format_converter import BaseFormatConverter, ConverterReturn
 from ._nlp_common import get_tokenizer, CLS_ID, SEP_ID
 
+def _new_check_is_max_context(doc_spans, cur_span_index, position):
```

Comment:
I do not see the difference from old _is_max_context, only doc_spans is dict, while in old ti si named tuple